routes/post.js

```diff
@@ -21,6 +21,14 @@ router.post(
   postController.createComment
 );
 
+router.post(
+  "/:postId/love",
+  tokenMiddleware.checkToken,
+  postController.validate("createPost"),
```

Comment:
I think we don't need any validator here. We are just taking no `body` in this POST request, right?

Reply:
Yes you're right, Pushed again @Ananto30